myapp/src/App.js

```diff
@@ -0,0 +1,190 @@
+import logo from './logo.svg';
+import beverly from './Images/beverly.jpeg'
+import fall from './Images/fall.jpeg'
+import plants from './Images/plants.jpeg'
+import self from './Images/self.jpeg'
+import './App.css';
+
+function App() {
+
+ 
+  /*
+  //comment out the map and objects for now since they aren't necessary for the website
+  //this was just for testing out a map with objects
+  const maria = {
+    name: "maria",
+    age: 19,
+    major: "computer science",
+    location: "massachusetts"
+  }
+
+  const nonExistentSibling = {
+    name: "matthew",
+    age: 22,
+    major: "biology",
+    location: "new york"
+
+  }
+
+  const kevin = {
+    name: "kevin",
+    age: 19,
+    major: "mechanical engineering",
+    location: "massachusetts"
+  }
+
+  const sarah = {
+    name: "sarah",
+    age: 19,
+    major: "english",
+    location: "delaware"
+  }
+
+  //create a map with objects
+  const friends_and_family = [nonExistentSibling, kevin, sarah];
+  const friendMap = friends_and_family.map((friend) =>{
+   
+    return (
+      
+      <h1>{friend.name}</h1>
+    )
+  });
+
+  */
+
+  //create a map
+  const foodsList = ["Apple Pie", "Pasta Salad", "Omelettes" ];
+  const foodMap = foodsList.map((food) =>{
+    return(
+      <p className="foodMapText">{food}</p>
+    )
+  });
+
+
+  //function to do something when button is clicked, have different methods for each image
+  const bevImgFunction = () => {
+    var x = document.getElementById("bevButton");
+
+    if (x.style.display === "none") {
+      x.style.display = "block";
+    } 
+    else {
+      x.style.display = "none";
+    }
+  }
+
+  const fallImgFunction = () => {
+    var x = document.getElementById("fallButton");
+    if (x.style.display === "none") {
+      x.style.display = "block";
+    } 
+    else {
+      x.style.display = "none";
+    }
+
+  }
+
+  const plantImgFunction = () => {
+    var x = document.getElementById("plantButton");
+    if (x.style.display === "none") {
+      x.style.display = "block";
+    } 
+    else {
+      x.style.display = "none";
+    }
+
+  }
+
+
+  //keeps track of characters typed into input box
+  const onChangeFunction = (event) => {
+    console.log(event.target.value);
+    var x = document.getElementById("textBoxAfterEnter");
+    console.log(x);
+    document.getElementById('textBox').onkeypress = function(e){
+      if (!e) e = window.event;
+      var keyCode = e.code || e.key;
+      if (keyCode == 'Enter'){
+        // Enter pressed
+        x.style.display = "block"; 
+        //when enter is pressed, display text that says "thanks for replying"
+        console.log("enter pressed");
+        
+      }
+    }
+  }
+ 
+  
+  return (
+    <div className="App">
+
+      <title>Maria Cristoforo's Website</title>
```

Comment:
so unfortunately `<title>` tags only work inside `<head>` tags, which don't go in the react return function. To change the title or the favicon you should put it in `public/index.html` (ill put a comment there so you can see where it goes)